src/Diagnostics.DataProviders/DataProviderConfigurations/GeoMasterDataProviderConfiguration.cs

```diff
@@ -0,0 +1,34 @@
+﻿
+namespace Diagnostics.DataProviders
+{
+    [DataSourceConfiguration("GeoMaster")]
+    public class GeoMasterDataProviderConfiguration : IDataProviderConfiguration
+    {
+        
+        public void PostInitialize()
+        {
+        }
+
+        public GeoMasterDataProviderConfiguration()
+        {
+        }
+
+        /// <summary>
+        /// GeomasterCertThumbprint
+        /// </summary>
+        [ConfigurationName("GeoRegionCertThumbprint")]
+        public string GeoRegionCertThumbprint { get; set; }
+
+        /// <summary>
+        /// GeomasterEndpoint
+        /// </summary>
+        [ConfigurationName("GeoEndpointAddress")]
+        public string GeoEndpointAddress { get; set; }
+
+        /// <summary>
+        /// IsInternal
+        /// </summary>
+        [ConfigurationName("Token")]
+        public string Token { get; set; }
```

Comment:
Do you have a flag to control ARM vs Geomaster call for private testing?

Reply:
I guess my question how do I switch between Geomasterclient and ArmClient

Reply:
@ShekharGupta1988 , he is checking to see if the GeoCertThumprint is set and if not then he uses GeoMasterTokenClient (ArmClient)

Reply:
https://github.com/Azure/Azure-AppServices-Diagnostics/pull/18/files/446c5c618a47f0d83f2d0712316fb5bf84258d1b#diff-dfb9d8e7d9a336d190de41b69312b210

Reply:
Yup, as Hawk said :-)